src/you_get/extractors/youku.py

```diff
@@ -237,7 +237,7 @@ def extract(self, **kwargs):
                 fileid    = fileid,
                 q         = q
             )
-            ksegs.append(u)
+            ksegs.append(json.loads(get_html(u+"&yxon=1"))[0]['server'])
```

Comment:
1. `get_html` is deprecated; we prefer to use `get_content` in future improvements.
2. Better style to encode the extra parameter `yxon` in the dict for `q`, instead of inserting as a string.